src/admin/components/AddAttendanceForm/style.less

```diff
@@ -78,16 +83,17 @@
     }
 
     .add-new-attendee {
-      background: @white;
+      background: var(--theme-background);
+      color: var(--theme-text-on-background-2);
```

Comment:
is this needed? i'd expect existing styles to not break if it's strictly refactoring

Reply:
Tried commenting out. Required for attendee tag to match the background.